### ISSUE TITLE
Avoid repackaging if requested

### ISSUE THREAD

**Issue by ItachiSan** [openSUSE/obs-service-tar_scm#110]
As in title.

**TL;DR**: An option to avoid tar repackaging would be nice.

**The long story**
In some PKGBUILDs (Arch Linux build description file), SCM sources get downloaded without repackaging, as the build system (*makepkg*) take care of download and else.

For download, OBS doesn't allow download during build time, and that's fine (we can just use a source service like this :stuck_out_tongue_winking_eye:), but the problem is that almost every service repackage the sources, at least in tar format.
This last thing is not good for Arch standards, as for SCM *makepkg* expects to download sources in a folder and create a clean source folder from it.

Having SCM sources repackaged requires packagers to change PKGBUILDs in order to track them.
This isn't a big fuss, however it breaks the usual build style of Arch packages.

A possible implementation would be to add a value in the service file for checking if repackaging is needed, in order to avoid it in case.
Default can be setted as `true`, to keep actual build consistent.

**Comment by aspiers** [comment]
I'm not sure I understand this 100% but it seems like a reasonable feature request. I'm afraid I won't have time to implement it, but pull requests are welcome.
